code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -47,6 +49,21 @@
 		to_chat(user, "<span class='userdanger'>Your body is wracked with debilitating pain!</span>")
 		return
 
+	if(optional)
+		if(!M.client)

```

Comment:
This isn't really checked otherwise. I'm not sure why, but they need to have a client for the alerts to work properly, so it is checked here. ¯\_(ツ)_/¯